generator/csharp.py

```diff
@@ -1,5 +1,3 @@
-from __future__ import unicode_literals
```

Comment:
Is there functionality changes here or is it all just python3 compatibility changes?

Reply:
I did remove the .csproj generation functionality as we want to be manually writing them using the new format going forwards. Other than that, just compat stuff.